### ISSUE TITLE
[Java] Avoid Empty Catch Block rule is violated despite allowing comments in rule. 

### ISSUE THREAD

**Issue by Fernal73** [pmd/pmd#1904]
**Affects PMD Version:** 

6.15, 6.16

**Rule:**
EmptyCatchBlock 
**Description:**
The rule is triggered despite a comment inserted in the empty catch block. 

**Code Sample demonstrating the issue:**

```
package networking;

import java.io.IOException;
import java.net.Socket;
import java.net.UnknownHostException;

public final class LowPortScanner {

  private LowPortScanner() {
    throw new IllegalStateException("Private constructor");
  }

  public static void main(String[] args) {
    String[] hosts = args.length > 0 ? args : new String[] {"localhost"};
    for (String host : hosts) {
      for (int i = 1; i < 1024; i++) {
        try {
          Socket s = new Socket(host, i);
          System.out.println("There is a server on port " + i + " of " + host);
          s.close();
        } catch (UnknownHostException ex) {
          System.err.println(ex);
          break;
        } catch (IOException ex) {  
          // must not be a server on this port
        }
      }
    }
  }
}

```

**Running PMD through:** *[CLI | Ant ]*

JDK Openjdk 10 on Arch Linux on Termux but compiling for Java 9.

The rule is configured as follows:

&lt;rule ref="category/java/errorprone.xml/EmptyCatchBlock"&gt;
    &lt;properties&gt;
        &lt;property name="allowCommentedBlocks" value="true" /&gt;
    &lt;/properties&gt;
&lt;/rule&gt;

**Comment by jsotuyod** [comment]
@Fernal73 I'm afraid I can't reproduce this with the given code snippet. I'm getting 0 violations when setting `allowCommentedBlocks` to `true`

**Comment by Fernal73** [comment]
My bad. Insert a copy of the above rule following the above as:

<rule ref="category/java/errorprone.xml/EmptyCatchBlock"/>

It runs the second instance of the rule producing the error. There is no checking the XML if there are duplicate rules in the file.
